src/scripts/main.js

```diff
@@ -1 +1,23 @@
 'use strict';
+
+const root = document.querySelector('.gallery');
+const largeImg = root.querySelector('#largeImg');
+const galleryList = root.querySelector('.gallery__list');
+const links = root.querySelectorAll('.list-item__link');
+
+for (const link of links) {
+  link.addEventListener('click', function(event_) {
+    event_.preventDefault();
+  });
+}
+
+galleryList.addEventListener('click', (event_) => {
```

Comment:
```suggestion
  link.addEventListener('click', function(event) {
    event_.preventDefault();
  });
}

galleryList.addEventListener('click', (event) => {
```
why do you have underscore at the end of event argument?

Reply:
Because when I use even I get this problem: 'event' is already declared in the upper scope.